web/packages/teleport/src/components/UserMenuNav/UserMenuNav.tsx

```diff
@@ -120,6 +122,7 @@ export function UserMenuNav({ username }: UserMenuNavProps) {
   const { preferences, updatePreferences } = useUser();
 
   const ref = useRefClickOutside<HTMLDivElement>({ open, setOpen });
```

Comment:
Could you rename `ref` to `outsideClickRef`? I feel like `ref` is not the best name for this var since the whole behavior behind detecting outside clicks rests on it, but it's named as if it was used just to caputer some random HTML element.